config/routes.rb

```diff
@@ -520,6 +520,7 @@ def patch(*) end # Disable PATCH requests
       get "#{root_path}/:username/badges" => "users#badges", constraints: { username: RouteFormat.username }
       get "#{root_path}/:username/bookmarks" => "users#bookmarks", constraints: { username: RouteFormat.username, format: /(json|ics)/ }
       get "#{root_path}/:username/user-menu-bookmarks" => "users#user_menu_bookmarks", constraints: { username: RouteFormat.username, format: :json }
+      get "#{root_path}/:username/user-menu-messages" => "users#user_menu_messages", constraints: { username: RouteFormat.username, format: :json }
```

Comment:
```suggestion
      get "#{root_path}/:username/user-menu-private-messages" => "users#user_menu_messages", constraints: { username: RouteFormat.username, format: :json }
```

It seems like all of our existing routes refer to PMs as private messages in the URL.